src/content/en/tools/chrome-devtools/profile/memory-problems/memory-diagnosis.markdown

```diff
@@ -1,197 +1,308 @@
-notes:
-  memory-terminology:
-    - "New to memory management? Get started with the basics in <a href='/web/tools/chrome-devtools/profile/memory-problems/memory-101'>Memory Terminology</a>."
+    - Find out how much memory your page is currently using with the 
+      Chrome Task Manager.
+    - Visualize memory usage over time with Timeline recordings.
+    - Identify detached DOM trees (a common cause of memory leaks) with
+      Heap Snapshots.
+    - Find out when new memory is being allocated in your JS heap with
+      Allocation Timeline recordings.
 ---
 
-<p class="intro">
-  Effective memory management is crucial for performance. Similar to native applications, web apps can suffer from memory leaks and bloat, but they also have to deal with garbage collection pauses.
-</p>
-
-A memory leak is a gradual loss of available computer memory.
-It occurs when a program repeatedly fails to return memory it has obtained for temporary use.
-When you think you have a memory leak,
-follow the investigative steps outlined in the table of contents.
+<p class="intro">Learn how to use Chrome and DevTools to find memory issues
+that affect page performance, including memory leaks, memory bloat, and
+frequent garbage collections.</p>
 
 {% include shared/toc.liquid %}
 
 {% include shared/takeaway.liquid list=page.key-takeaways.too-much-memory %}
 
-## Check if page uses too much memory
-
-To check if a page uses too much memory, the first thing to do is identify a sequence of actions you suspect is leaking memory. 
-This could be anything from navigating around a site, hovering, clicking, or otherwise somehow interacting with page in a way that seems to negatively impact performance more over time.
-
-Once you suspect memory performance issues,
-use the
-[Chrome DevTools Timeline](/web/tools/chrome-devtools/profile/evaluate-performance/timeline-tool)
-to diagnose excessive memory usage
-when you first notice your page has slowed down after extended use.
-
-{% include shared/remember.liquid title="Note" list=page.notes.memory-terminology %}
-
-### Monitor memory using Chrome task manager
+## Overview
 
-Monitor a page's live memory usage using the Chrome task manager.
+In the spirit of the [RAIL][RAIL] performance model, the focus of your
+performance efforts should be your users.
 
-Access the Task Manager from the Chrome menu > Tools > Task Manager or by pressing <span class="kbd">Shift</span> + <span class="kbd">Esc</span>.
+Memory issues are important because they are often
+perceivable by users. Users can perceive memory issues in the following
+ways:
 
-Once open, right-click on the heading area of the columns and enable the JavaScript memory column.
-Perform actions that may use too much memory and monitor how the live memory usage changes:
+* A page's performance gets progressively worse over time. This is possibly
```

Comment:
I think this section would benefit from bolding the first sentences in the list.